README.md

```diff
@@ -70,7 +70,7 @@ This will display all devices detected by fwupd.
 
 `# fwupdmgr refresh`
 
-This will download the latest metadata from LVFS.
+This will download the latest metadata from LVFS, and start serving the cache on `0.0.0.0:27500`. Other clients on the same network will make use of this via mDNS/LLMNR.
```

Comment:
I think this isn't 100% correct -- many something like this instead?:

    This will download the latest metadata from LVFS. If the [Passim](https://github.com/hughsie/passim/blob/main/README.md) project is also installed and enabled, fwupd will re-publish the metadata file which will be served on `0.0.0.0:27500` by default. Other clients on the same network can make use of this via mDNS/LLMNR to reduce network bandwidth.

Reply:
even better!

I would make it simpler/shorter removing the conditional for passim. Currently a built time option and everyone building from *this* source's default will have it, A distro that change the build can describe it any way they like for their distribution choices.

Reply:
another suggestion, replaced the verbosity on optional passim for verbosity about where are the relevant configs:
 
> This will download the latest metadata from LVFS. fwupd will use [Passim](https://github.com/hughsie/passim/blob/main/README.md) to re-publish the metadata  file (`P2pPolicy=metadata` under `[fwupd]` in `fwupd.conf`)  which will be served on `0.0.0.0:27500` by default (see passim configuration for port change). Other clients on the same network can make use of this via mDNS/LLMNR to reduce network bandwidth.

Reply:
> Currently a built time option and everyone building from this source's default will have it

Right, but implying it's compulsory is going to attract the wrong type of drama.

Reply:
I think I see what you mean. But not mentioning it is more likely to create the drama IMO.

Reply:
@hughsie I am wondering if we actually want to make passim a prompted question and change the default to disabled?  It will avoid confusion and give more obvious "control" to a user.

Reply:
> make passim a prompted question

In meson? or in fwupdmgr?

Reply:
`fwupdmgr`.

IoW change the default in `fwupd.conf` to have P2P metadata turned off and then as part of `fwupdmgr` commands prompt to share either the metadata or the payload data.

Reply:
i think those are fine, but let's just document current behavior first, please?

Reply:
Alternate proposal in https://github.com/fwupd/fwupd/pull/7261 -- I'm quite keen Passim is enabled by default otherwise it's order-of-magnitude less effective in the real world.

Reply:
btw, i see on the data collection code https://github.com/fwupd/fwupd/blob/d181f36c1d9a12d7e617e25907056adb1b0038eb/src/fu-engine.c#L1716C8-L1716C41 is then logging all downloads https://github.com/hughsie/passim/blob/ae38c13da1a63fff8c8fa0b8e343ecb1cd417e5a/src/passim-server.c#L1017

i don't know how that value is used, but reason I came here to add to this note is because we had `fwupd` on a firewall appliance we were testing and the passim port happened to be open. We saw lots of unwanted traffic on that lab. not sure if invalid of random requests get counted on that.

Reply:
> Surely intranet traffic is better than internet traffic?

I wish. External traffic. Not all machines are behind NAT.